src/com/readers/SudokuReader.java

```diff
@@ -0,0 +1,65 @@
+package com.readers;
+
+import com.sudoku.SudokuBoard;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Class that contains readers methods to read Sudokus from files
+ *
+ * @author Jose Cabrera
+ */
+public class SudokuReader {
+    protected String filePath;
+    protected Scanner scanner = null;
+    public static final int ASCCI_ZERO = "0".charAt(0);
+    private static int BOARD_SIZE = 9;
+    private File file;
+    public SudokuReader(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
+
+    /**
+     * Method to return a SudokuBoard
+     *
+     * @return SudokuBoard is a sudokuBoard that contains the data obtained from the file
+     */
+    public SudokuBoard getSudokuBoard() {
+        String parsedText = read().replace(",", "");
+        int iterator = 0;
+        int[][] grid = new int[BOARD_SIZE][BOARD_SIZE];
+        for (int col = 0; col < BOARD_SIZE; col++) {
+            for (int row = 0; row < BOARD_SIZE; row++) {
+                grid[col][row] = parsedText.charAt(iterator) - ASCCI_ZERO;
+                iterator++;
+            }
+        }
+        return new SudokuBoard(grid);
+    }
+
+    /**
+     * Read a file
+     *
+     * @return text obtained from file
+     */
+    public String read() {
```

Comment:
correct me if am wrong but this method should be private right, this should be only used in the getSudokuBoard?